bedrock/firefox/templates/firefox/includes/download-button-thanks.html

```diff
@@ -0,0 +1,18 @@
+<div id="{{ id }}" class="mzp-c-button-download-container c-button-download-thanks">
+  <a href="{{ transition_url }}"
+     class="download-link mzp-c-button mzp-t-product {% if button_class %}{{ button_class }}{% else %}mzp-t-xl{% endif %}"
+     data-direct-link="{{ download_link_direct }}"
+     data-link-type="download"
```

Comment:
There's a change in the GA reporting. This is showing up with the Event Label "Download for unavailable". I think it is because `data-download-os` is defined on the old button but not this one.

Reply:
Otherwise GA reporting looks to be the same in Chrome/MacOS:

| Prod     | Demo |
| --- | --- |
| ![Screen Shot 2020-09-15 at 07 22 12](https://user-images.githubusercontent.com/854701/93225064-a9ee3500-f726-11ea-9b7a-d837ce323814.png) | ![Screen Shot 2020-09-15 at 07 24 19](https://user-images.githubusercontent.com/854701/93225083-ad81bc00-f726-11ea-84de-02bf641d5d9a.png) |

Reply:
Strange, because I'm adding `data-download-os` via JS (since the value is dynamic)

Reply:
I can see `data-download-os` on the link when viewing source on demo:

<img width="1674" alt="image" src="https://user-images.githubusercontent.com/400117/93226105-de83e000-f76a-11ea-8039-79029a54415b.png">

Reply:
Oh wait - it's `data-display-os` (face palm). At least that's a sinple fix :)

Reply:
Oh, I was viewing source. Then I don't know what's causing the difference. Are you able to look into it?

Reply:
Yeah I think I've got it - typo (see comment above ^) :)

Reply:
LOL, I made the same mistake in my comment 🤦‍♀️

Reply:
Updated data attribute to the correct name